packages/lib/Synchronizer.ts

```diff
@@ -193,15 +192,62 @@ export default class Synchronizer {
 		return `${duration}ms`;
 	}
 
-	// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
-	public static reportToLines(report: any) {
+	public static reportToLines(report: SyncReport) {
+		const formatItemCounts = (counts: ItemCountPerType) => {
+			const modifiedItemNames: string[] = [];
+			let hasOther = false;
+
+			let sum = 0;
+			for (const [key, value] of Object.entries(counts)) {
+				if (value) {
+					sum += value;
+
+					if (key === 'Revision') {
+						modifiedItemNames.push(_('note history'));
+					} else if (key === 'Note') {
+						modifiedItemNames.push(_('notes'));
+					} else if (key === 'Resource') {
+						modifiedItemNames.push(_('resources'));
+					} else if (key === 'Folder') {
+						modifiedItemNames.push(_('notebooks'));
+					} else {
+						hasOther = true;
+					}
+				}
+			}
+
+			// In some cases, no type information is available (e.g. when creating local items).
+			// In these cases, avoid logging "other", because that might be inaccurate.
+			if (hasOther && modifiedItemNames.length > 0) {
+				modifiedItemNames.push(_('other'));
+			}
+
+			if (modifiedItemNames.length > 0) {
+				return _('%d (%s)', sum, modifiedItemNames.join(', '));
```

Comment:
As mentioned in the pull request description, `modifiedItemNames.join(', ')` isn't ideal for localization purposes.